main/src/ca/uwaterloo/flix/language/phase/jvm/JvmBackend.scala

```diff
@@ -65,6 +65,7 @@ object JvmBackend {
       val erasedLazyTypes = JvmOps.getErasedLazyTypesOf(types)
       val erasedExtendTypes = JvmOps.getErasedRecordExtendsOf(types)
       val erasedFunctionTypes = JvmOps.getErasedArrowsOf(types)
+      val erasedTuplesTypes = JvmOps.getErasedTupleTypesOf(types)
```

Comment:
All of these might end up being expensive. Perhaps we need to do just one iteration at some point.

Reply:
These are filtered and then genClass is called on all of them. Soon (or now?) it should be `types.map(genClass)` where genClass figures out what to do depending on the type